app/uk/gov/hmrc/helptosavefrontend/controllers/EnrolmentCheckBehaviour.scala

```diff
@@ -48,11 +49,13 @@ trait EnrolmentCheckBehaviour {
         // if the user is enrolled but the itmp flag is not set then just
         // start the process to set the itmp flag here without worrying about the result
         if (!itmpHtSFlag) {
-          helpToSaveService.setITMPFlag(nino).fold(
-            e ⇒ logger.warn(s"Could not start process to set ITMP flag for user $nino: $e"),
-            _ ⇒ logger.info(s"Process started to set ITMP flag for user $nino")
-          )
+          helpToSaveService.setITMPFlag(nino).value.onComplete{
+            case Failure(e)        ⇒ logger.warn(s"Could not start process to set ITMP flag for user $nino, future failed: $e")
```

Comment:
For all NINO log recordings, pls can you start the msg "For NINO, xxxx" as this will ensure all our log msgs are consistent in this respect, to help any post-live support

Reply:
sure - but aren't we going to remove all NINO logging in the frontend at some point? Or are we not doing that anymore?